django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
Did you see that `TimeField` uses `datetime.datetime.now().time()` and `DateField` `datetime.date.today()`? I guess there might be a discrepancy with using `timezone.now()` in some cases.